src/bin/nydus-image/main.rs

```diff
@@ -529,6 +534,36 @@ fn prepare_cmd_args(bti_string: &'static str) -> App {
             .arg(arg_output_json.clone()),
     );
 
+    let app = app.subcommand(
+        App::new("optimize")
+            .about("Optimize By Prefetch")
+            .arg(
+                Arg::new("bootstrap")
+                    .help("File path of RAFS metadata")
+                    .short('B')
+                    .long("bootstrap")
+                    .required(true),
+            )
+            .arg(
+                Arg::new("prefetch-files")
+                    .long("prefetch-files")
+                    .short('p')
+                    .help("Prefetch files")
+                    .action(ArgAction::Set)
+                    .num_args(1),
+            )
+            .arg(arg_config.clone())
+            .arg(
+                Arg::new("blob-dir")
```

Comment:
Should use `--backend-type`, `--backend-config`, `--backend-config-file` like `compact` subcommand.